### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN039.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#86] [General]
# https://blog.3place.cn/3place/lmyn/LMYN039.html

https://blog.3place.cn/3place/lmyn/LMYN039.html

**Comment by picklecai** [comment]
祭祀者确定了时间，
在这一段时间里，
因陀罗变成女罗刹，
把那匹祭马偷了去。 （1.38.7）

因陀罗为什么这么调皮？不是天帝吗？

**Comment by picklecai** [comment]
父亲的指令是：
一由旬一由旬地走向前去。 （1.38.14）
只要有那匹马的影子，
你们就要把大地来挖。

儿子们的实践是：
他们遵照父命，
走遍了整个大地。 （1.38.17）
他们中的每一个人，
都把大地挖进一由旬。
……
这些英雄们，
一直在挖掘着大地，
挖下去了六万由旬，
直挖到最深层地狱。 （1.38.21）

挖地三尺也要找出来，变成了这样。
而且儿子们是高高兴兴地接受任务的：

身强力壮的王子们，
听了这话心里欢喜。
